src/main/java/edu/ucla/library/iiif/fester/verticles/ManifestVerticle.java

```diff
@@ -357,23 +357,23 @@ private void updateWorks(final CsvHeaders aCsvHeaders, final CsvMetadata aCsvMet
         final Promise<LockedManifest> promise = Promise.promise();
         final String collectionID = aCsvMetadata.getFirstCollectionID(aCsvHeaders.getParentArkIndex()).get();
 
-        // If we were able to get a lock on the manifest, update it with our new works
+        // If we were able to get a lock on the collection, update it with our new works
         promise.future().onComplete(handler -> {
             if (handler.succeeded()) {
                 final Map<String, List<String[]>> worksMap = aCsvMetadata.getWorksMap();
-                final LockedManifest lockedManifest = handler.result();
+                final LockedManifest lockedCollection = handler.result();
```

Comment:
No objections to this, but isn't it kind of a superficial change if we keep `LockedManifest` and `getLockedManifest()`? The distinction we're making is in variable name only. I'm wondering if you'd see this change as a stepping stone to something more comprehensive(?)

Reply:
Maybe this method should be renamed as well? `getLockedIiifResource`? In general I'm interested in making the manifest/collection distinction more clear throughout the codebase.

Reply:
And LockedManifest to LockedIiifResource?

Reply:
Makes sense to me 👍🏼

Reply:
I do wonder whether the `Iiif` part is necessary... is LockedResource too generic (even in the context of the app)?

Reply:
I don't think it makes a huge difference, but I do think the `Iiif` part helps me more easily remember what the method/class does/is when I come across it in the code